interactions/api/dispatch.py

```diff
@@ -27,12 +27,9 @@ def dispatch(self, __name: str, *args, **kwargs) -> None:
         r"""
         Dispatches an event given out by the gateway.
 
-        :param __name: The name of the event to dispatch.
-        :type __name: str
-        :param *args: Multiple arguments of the coroutine.
-        :type *args: list[Any]
-        :param **kwargs: Keyword-only arguments of the coroutine.
-        :type **kwargs: dict
+        :param str __name: The name of the event to dispatch.
+        :param list[Any] \*args: Multiple arguments of the coroutine.
```

Comment:
`*args` is an `Any` and `args` is a `Tuple[Any]`, not a list. Same think for `kwargs`